addon/utils.js

```diff
@@ -54,3 +54,17 @@ export function createCollection(collection, alias) {
     return create.bind(this)(model, alias);
   }.bind(this));
 }
+
+/*
+ * Decorate collection or Object
+ * @param forDecoration[Array/Object]
+ * @param alias[String]
+ * @return Array/Object
+ */
+export function decorate(forDecoration, alias) {
+  if (Ember.typeOf(forDecoration) === 'array') {
```

Comment:
Not sure which is more accurate, although there is `Ember.isArray(forDecoration)` which seems a bit more concise